Common/src/gRPC/Services/GrpcTasksService.cs

```diff
@@ -121,4 +120,44 @@ public class GrpcTasksService : Tasks.TasksBase
                                         "Unknown Exception, see application logs"));
     }
   }
+
+  public override async Task<GetResultIdsResponse> GetResultIds(GetResultIdsRequest request,
+                                                                ServerCallContext   context)
+  {
+    try
+    {
+      return new GetResultIdsResponse
+             {
+               TaskResult =
+               {
+                 await taskTable_.GetTasksExpectedOutputKeys(request.TaskId,
+                                                             context.CancellationToken)
+                                 .Select(r => new GetResultIdsResponse.Types.MapTaskResult
+                                              {
+                                                ResultIds =
+                                                {
+                                                  r.expectedOutputKeys,
+                                                },
+                                                TaskId = r.taskId,
```

Comment:
Just for coherence with the object name, I'd put the TaskId before the ResultIds